src/routes.tsx

```diff
@@ -20,6 +21,7 @@ const routes = [
       { path: 'customers', element: <CustomerListView /> },
       { path: 'dashboard', element: <DashboardView /> },
       { path: 'products', element: <ProductListView /> },
+      { path: 'hello', element: <HelloWorld />},
```

Comment:
funny thing, we already have routing in this example/project

I will reach Matej, if we need new row for the next demo (it depends on his technical solution)